packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -149,6 +166,7 @@ class ThemeData extends Diagnosticable {
     MaterialTapTargetSize materialTapTargetSize,
     PageTransitionsTheme pageTransitionsTheme,
     ColorScheme colorScheme,
+    SnackBarBehaviour snackBarBehaviour,
```

Comment:
Make sure to add checks for snackBarBehaviour in the `==`, 'hashCode', and 'debugFillProperties'